merlin-server/sql/merlin/tables/metadata/users.sql

```diff
@@ -0,0 +1,18 @@
+create table metadata.users(
+  username text not null primary key,
+  default_role text not null references metadata.user_roles
+    on update cascade
+    on delete restrict
+);
+-- Insert the default roles into the table, then change the generated status to "Always"
+-- This can be changed back if we need to add more default users in the future
+insert into metadata.users(username, default_role)
+  values ('Mission Model', 'viewer'),
```

Comment:
What is the meaning of these default usernames?

Reply:
Does the `viewer` part have any significance?

Reply:
Somewhat. `default_role` is a non-nullable column, and `viewer` is the most restrictive role. However, even if a user managed to log in as a "reserved" user, they wouldn't be able to perform any queries/mutations, since they have no `allowed_roles`.